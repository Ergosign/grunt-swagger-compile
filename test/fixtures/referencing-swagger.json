{
    "swagger": "2.0",
    "info": {
        "version": "1.0.1",
        "title": "Test Swagger"
    },
    "paths":{
        "/test":{
            "get":{
                "$ref": "sub-directory/referenced.json"
            }
        }
    }
}
