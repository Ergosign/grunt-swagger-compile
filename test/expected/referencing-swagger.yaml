swagger: '2.0'
info:
  version: 1.0.1
  title: Test Swagger
paths:
  /test:
    get:
      description: Returns all pets from the system that the user has access to
      produces:
        - application/json
      responses:
        '200':
          description: A list of pets.
          schema:
            type: array
            items:
              $ref: '#/definitions/pet'
